eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this a necessary / related change?

Reply:
Good catch. This was an accidental change. Fixed it.